### PR TITLE
Add introduction and prerequisites

### LINKED ISSUE #17: Move over course overview from Chamilo

### PR DESCRIPTION
**Pull request by thomasaelbrecht**

### CODE REVIEW

File: docs/00-introduction.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - [Markdown All in One (Yu Zhang)](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)
+    - [markdownlint (David Anson)](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
+- **Git client** (including Git Bash on Windows). A graphical Git client like Gitkraken is optional (and *not* recommended).
+- **Docker** (including Docker Compose). You can install Docker Desktop for Windows or Mac, or Docker CE for Linux. Use WSL2 as the backend on Windows.
+
+It is best to install the required software using a package manager, e.g., [Chocolatey](http://chocolatey.org/) on Windows or [Homebrew](https://brew.sh/) for Mac. Linux users are, of course, familiar with this method and use their distribution's package manager, e.g., `apt` (Debian, Ubuntu), `dnf` (Fedora and RHEL family), `pacman` (Arch Linux), etc. Check if you have these tools installed. If so, upgrade to the latest versions. If not, install them through the package manager.
+
+:information_source: For this course, the use of GitHub Copilot is allowed.
+
+Hier is de vertaling van de HTML-pagina naar het Engels en omgezet in Markdown:
```
Comment (MartenBE) [inline docs/00-introduction.md]: This is probably a left over?

File: docs/00-introduction.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Guidelines for starting your Git project:
+
+- You should already have a GitHub account. If you haven't done so yet, be sure to link your `@student.hogent.be` email address to the account. You can then apply for the [**GitHub Student Developer Pack**](https://education.github.com/pack) which includes various interesting offers. It's possible to register multiple email addresses, and it makes sense to also link your private address. This way, you can continue using your GitHub account after graduation.
+
+- Create an [**SSH key pair**](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to simplify pushing to GitHub, follow the instructions provided. For convenience, enter an empty passphrase (otherwise, you'll need to enter the passphrase every time you use the key).
+
+- Enable [**two-factor authentication**](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) on your GitHub account. This is a security measure that is now required on GitHub.
+
+- **Basic Git configuration**, if you haven't done this before (check in the configuration file `~/.gitconfig`):
```
Comment (MartenBE) [inline docs/00-introduction.md]: The XDG specifications specify `$HOME/.config/` as the place to be for config files: https://wiki.archlinux.org/title/XDG_Base_Directory.
E.g. for git the config file would be `$HOME/.config/git/config`.

File: docs/00-introduction.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Create a **local directory** that you reserve for everything related to this course. Within this directory, you can clone your GitHub repository. On your repository's GitHub page, click on the green button on the right (Code), choose "Use SSH", and copy the link in the format `git@github.com:HoGentTIN/REPO_NAME-USERNAME.git`. This will create a local copy of the repository in a subdirectory. You can rename this directory and move it; everything will still work. It's a bad idea to clone the repository in a folder that is synced to the cloud (e.g., OneDrive, Dropbox, Google Drive, iCloud, etc.).
+
+```bash
+cd Documents/Courses/MLOps/
+git clone git@github.com:HOGENT-MLOps/REPO_NAME-USERNAME.git
+```
+
+- Review the files in the `assignments` directory. Here you'll find the assignments for all labs.
+- In the `reports` directory, you'll find templates for reports and cheat sheets in [Markdown format](https://guides.github.com/features/mastering-markdown/). Customize the template by adding your name.
+    - There is an example assignment `00-dummy.md` with an example report `00-dummy-report.md`. Have a look at them to see how you are expected to structure your lab reports.
```
Comment (MartenBE) [inline docs/00-introduction.md]: Perhaps mention that this is the minimum we expect?

File: docs/00-introduction.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- Also, immediately add your name and repository URL to the `README.md`.
+
+## Processing errata
+
+It is possible that errors may be found in the lab assignments during the semester. These will be published in the [source repository](https://github.com/HOGENT-MLOps/mlops-labs). You can systematically apply these changes to your own personal GitHub repository where you are working on the lab assignments. Simply open your GitHub repository in the browser and click the "Sync" button.
+
+Any errata will be announced via Chamilo.
+Hier is de vertaling van de HTML-pagina naar het Engels en omgezet in Markdown:
```
Comment (MartenBE) [inline docs/00-introduction.md]: Probably a leftover?

File: docs/00-introduction.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+In this section, you will find some general guidelines to help you work more smoothly and efficiently.
+
+- Before starting a lab assignment, prepare yourself by studying all the provided study materials: manuals, screencasts, etc. There is no need to "memorize" everything, but make sure you are familiar enough with the content so that you can quickly find the correct and relevant information. You can find this through the references in the individual lessons on this page.
+
+- **Work step by step.** Don't write too much code at once. First, try to obtain a minimally working solution and immediately commit it to Git. Make minimal changes and test each change thoroughly. The larger and more significant the changes, the greater the chance of errors and the more difficult they are to debug. As soon as something works and you've made progress, commit it to Git right away with a clear, descriptive commit message.
+
+- **Learn to use Git from the command line.** Most Git commands provide detailed explanations on how to proceed and how to undo the last step. This will eventually give you a better understanding of how Git works than you would get from using a GUI.
```
Comment (MartenBE) [inline docs/00-introduction.md:69]: Perhaps we shouldn't advise Git GUI tools and plugins in the software section?

File: docs/00-introduction.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+Any errata will be announced via Chamilo.
+Hier is de vertaling van de HTML-pagina naar het Engels en omgezet in Markdown:
+
+## General Guidelines
+
+In this section, you will find some general guidelines to help you work more smoothly and efficiently.
+
+- Before starting a lab assignment, prepare yourself by studying all the provided study materials: manuals, screencasts, etc. There is no need to "memorize" everything, but make sure you are familiar enough with the content so that you can quickly find the correct and relevant information. You can find this through the references in the individual lessons on this page.
+
+- **Work step by step.** Don't write too much code at once. First, try to obtain a minimally working solution and immediately commit it to Git. Make minimal changes and test each change thoroughly. The larger and more significant the changes, the greater the chance of errors and the more difficult they are to debug. As soon as something works and you've made progress, commit it to Git right away with a clear, descriptive commit message.
```
Comment (MartenBE) [inline docs/00-introduction.md]: Specification for commit messages: https://www.conventionalcommits.org/en/v1.0.0/

**Review by MartenBE — CHANGES_REQUESTED** [review]

File: docs/00-introduction.md (hunk 1 above)
Comment (thomasaelbrecht) [reply docs/00-introduction.md]: Yes, ChatGPT helped me with the translation from HTML (Dutch + Chamilo) to English (markdown).

**Review by MartenBE — APPROVED** [review]

**Pull request merged**